run_lstm_check_results.ipynb

```diff
@@ -0,0 +1,739 @@
+{
+ "cells": [
```

Comment:
I think this file is on the .gitignore list and thus will not be packaged with the rest of the code in the data release. What value do you see in checking it into the github repo?

Reply:
I was working in this notebook while sorting out how to run the model code, and wanted to have something saved while that work was ongoing.  But now that the environment is sorted out and the readme is mostly done it is fairly expendable, so could be removed.